libbpf-tools/cgroup_helpers.c

```diff
@@ -0,0 +1,291 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+/* Copyright (c) 2025 Rong Tao */
+#include <assert.h>
+#include <dirent.h>
+#include <errno.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <string.h>
+#include "cgroup_helpers.h"
+
+
+int cgroup_get_roots(char ***roots)
```

Comment:
Looks like this should be a static function as it is only used in this file. The prototype in the cgroup_helpers.h is not needed. Please also check other functions.